utils/getInvoices/tests/getInvoices.discount.test.ts

```diff
@@ -206,9 +206,9 @@ describe('getInvoices - DISCOUNT', () => {
         company,
       })
 
-      expect(invoices).not.toContainEqual({
-        type: ServiceType.Discount,
-      })
+      expect(invoices).toContainEqual(
+        expect.objectContaining({ type: ServiceType.Discount })
+      )
```

Comment:
No `expect.objectContaining()` on 'shoud load' tests. We should receive exact result